.gitlab-ci-darwin.yml

```diff
@@ -0,0 +1,73 @@
+variables:
+  SCHEDULER_PARAMETERS: '--nodes=1 --partition=power9 --export=NONE'
+  GIT_SUBMODULE_STRATEGY: recursive
+  MODULE_CMAKE: "cmake/3.11.1"
+  MODULE_CLANG: "clang/8.0.1"
+  MODULE_COMPILER: "gcc/7.4.0"
+  MODULE_CUDA: "cuda/10.1"
+  MODULE_MPI: "openmpi/p9/4.0.2-gcc_7.4.0"
+  NUM_MPI_PROC_TESTING: 2
+  OMP_NUM_THREADS: 40
+  CMAKE_BUILD_TYPE: "Release" 
+  Kokkos_ARCH_POWER9: "ON" 
+  Kokkos_ARCH_VOLTA70: "ON" 
+  Kokkos_ENABLE_CUDA: "ON" 
+  Kokkos_ENABLE_CUDA_UVM: "OFF"
+  Kokkos_ENABLE_CXX11: "ON" 
+  Kokkos_ENABLE_OPENMP: "ON" 
+  PARTHENON_DISABLE_HDF5: "OFF" 
+
+stages:
+  - performance-regression
+
+# Is performed before the scripts in the stages step
+before_script:
+  - env -i bash ./scripts/darwin/setup.sh
+  - env -i bash ./scripts/darwin/build_hdf5_parallel.sh
+  - export GOLD_STANDARD_VER=2
+  - if [ `cat tst/regression/gold_standard/current_version` != ${GOLD_STANDARD_VER} ]; then
+    echo "Updating regression Gold standard to ${GOLD_STANDARD_VER}." &&
+    wget -qO- https://pgrete.de/dl/parthenon_regression_gold_r${GOLD_STANDARD_VER}.tgz | tar -xz -C tst/regression/gold_standard; fi
+
+parthenon-power9-gcc-mpi-performance-regression:
+  variables: 
+    BUILD_DIR: "build_power9_perf_regression_gcc_mpi"
+    CMAKE_CXX_COMPILER: $CI_PROJECT_DIR/external/Kokkos/bin/nvcc_wrapper
+  stage: performance-regression
+  when: manual
```

Comment:
instead of manual triggers, could we do a scheduled build, maybe nightly or something? I'd like an email from gitlab if we break the regression tests on power9s.

Reply:
I'd prefer to never break anything. The manual option should allow you to trigger any pull request before it is merged with master. It just requires that you have access to lanl gitlab to cause the ci to build. We could add a scheduled build but then you have already merged with master.

Reply:
Can we do both? I'm nervous about making it *only* manual. Because we're all only human. At some point, somebody is going to forget. And that might be the time it breaks.

Reply:
Fair point.

Reply:
> Can we do both? I'm nervous about making it _only_ manual. Because we're all only human. At some point, somebody is going to forget. And that might be the time it breaks.

The other option is to make the manual trigger blocking. As in you can't merge unless it is run.

Reply:
I'd be okay with that. I like that option less, though, because it's an extra barrier for external collaborators. And sometimes it won't make sense, like for documentation.

Reply:
> I'd be okay with that. I like that option less, though, because it's an extra barrier for external collaborators. And sometimes it won't make sense, like for documentation.

Good point.